haskell-ide-engine.cabal

```diff
@@ -84,6 +85,7 @@ library
                      , monoid-subclasses > 0.4
                      , mtl
                      , optparse-simple >= 0.0.3
+                     , ormolu
```

Comment:
Guard ormolu to only be included if the ghc version is greater than 8.4

Reply:
Done!